application-admintools-ui/src/main/resources/AdminTools/Code/Configuration.xml

```diff
@@ -51,21 +51,42 @@
       <defaultWeb/>
       <nameField/>
       <validationScript/>
+      <excludedLines>
```

Comment:
These changes are not not related to this PR. You should do a separate commit on main to add what you maybe forgot while working on other issues.
Same for what is related to mvn xar:format. I see you have a lot of `<xwikidoc version="1.5"` changes for example, but this will just make git history harder to check (you have some files changed as part of this issue, even though they are not related). So I suggest to put these xar:format changes in a separated commit on main (they are present in other PRs as well). 
I think that most of the time, if you exported from the right xwiki version, xar:format should not add extra changes (unless you indeed changed more than the content of the pages)

You will also make the PR easier to follow